javascripts/discourse/api-initializers/table-editor.js

```diff
@@ -59,7 +59,11 @@ export default apiInitializer("0.11.1", (api) => {
       if (table.parentNode.contains(expandBtn)) {
         expandBtn.parentNode.insertBefore(popupBtn, expandBtn);
       } else {
-        table.parentNode.insertBefore(popupBtn, table);
+        const buttonWrapper = document.createElement("div");
+        buttonWrapper.classList.add("fullscreen-table-wrapper-buttons");
+        buttonWrapper.append(popupBtn);
+        popupBtn.addEventListener("click", generateModal, false);
+        table.parentNode.insertBefore(buttonWrapper, table);
       }
 
       popupBtn.addEventListener("click", generateModal.bind(attrs), false);
```

Comment:
Not new to this PR, but ideally these event listeners need to be cleaned up. Can probably be done via `api.cleanupStream()`